templates/javascript/conditional_response.js

```diff
@@ -26,7 +26,7 @@ async function handleRequest(request) {
     return new Response('Block the IP 1.2.3.4', { status: 403 })
   }
   // On ASN
-  if ((request.cf || {}).asn == 64512) {
```

Comment:
Was there a reason for being paranoid about the existence of request.cf in the original snippet?

Reply:
it doesn't exist in our GCP preview worker, so yes. i think we should probably keep that conditional there.

Reply:
Brought it back 👍